4-WebApp-your-API/4-1-MyOrg/Client/Models/TodoItem.cs

```diff
@@ -1,5 +1,8 @@
-﻿namespace TodoListService.Models
+﻿namespace TodoListClient.Models
 {
+    /// <summary>
+    /// Data object to transfer information between client/server/database
+    /// </summary>
     public class Todo
```

Comment:
>Todo [](http://example.com/codeflow?start=17&length=4)

late comment, but can the model move to the service instead pls. In correct scenarios, the models/entities are published by the service along with operations and clients should be pure consumers..